### PR TITLE
new code after review (Introduction to GitHub)

### PR DESCRIPTION
**Pull request by williamaguessy**
I revised my code based on my classmate's suggestions.

### CODE REVIEW

File: Code review for waa94/codingame-1.c [hunk 1]
```diff
@@ -0,0 +1,136 @@
+// # Coding with reflexion
+
```
Comment (bjones1) [inline Code review for waa94/codingame-1.c:2]: Put a `//` for empty lines between comments.

File: Code review for waa94/codingame-1.c [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+// ## Power of Thor
+
+// _here his a link to CodinGame to use this solution :_
+// [CodinGame](https://www.codingame.com/)
+
+// **Here is the second version of my code following the review :** I have added
```
Comment (bjones1) [inline Code review for waa94/codingame-1.c:8]: Good, but remove this from your code -- it belong in the pull request discussion area.

File: Code review for waa94/codingame-1.c [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+// the link above, revised the paragraphs for a clearer explanation of my
+// errors, included a two-dimensional image of Thor's movement, corrected the
+// labeling issue, and added comments to better explain the conditions and
+// Thor's movement at each iteration.
+
+// ### Pre writting
+
```
Comment (bjones1) [inline Code review for waa94/codingame-1.c:15]: Extra blank line.

**Review by bjones1 — CHANGES_REQUESTED** [review]

**Comment by williamaguessy** [comment]
I have revised my code based on your recommendations. Should I submit another pull request?

**Comment by williamaguessy** [comment]
My Pull request is ready for review.

**Pull request merged**